language_detector/main.py

```diff
@@ -1,9 +1,16 @@
-"""This is the entry point of the program."""
-
+# -*- coding: utf-8 -*-
 from .languages import LANGUAGES
 
 
 def detect_language(text, languages=LANGUAGES):
-    """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    best_guess = ''
+    best_guess_count = 0
+    for current_language in LANGUAGES:
+        count = len([words for words in text.strip().split(' ') if words in LANGUAGES[current_language]])
```

Comment:
Instead of spiting the text string into a list of words every time you compare it against a language, I would do this once outside the for loop and save it for reuse. This would improve the performance of your code by not repeating this operation:
```
    text_list = text.strip().split(' ')
    for current_language in LANGUAGES:
        count = len([words for words in test_list if words in LANGUAGES[current_language]])
```